Mathlib/Topology/Sets/Opens.lean

```diff
@@ -306,6 +306,12 @@ theorem IsBasis.isCompact_open_iff_eq_finite_iUnion {ι : Type*} (b : ι → Ope
     simp
   · exact hb'
 
+lemma IsBasis.le_iff {α} {t₁ t₂ : TopologicalSpace α}
```

Comment:
(By the way, maybe a leaner syntax such as`IsBasis[t₂] Us` was preferable, I don't know the usages here.)

Reply:
I just looked for it as well but seems like it does not exist. This is the only occurrence of `@IsBasis` in mathlib so maybe it's not worth adding it.

Reply:
Maybe that exists for `Opens`, anyway, there's no ambiguity here which is what matters.